libraries/AP_Airspeed/AP_Airspeed_MS4525.cpp

```diff
@@ -47,6 +47,7 @@ bool AP_Airspeed_MS4525::init()
         uint8_t bus;
         uint8_t addr;
     } addresses[] = {
+        { 2, MS4525D0_I2C_ADDR },
```

Comment:
@lucasdemarchi Once you and @tridge solve #6265, we can change all drivers and libraries to use it 🙂 Until then, there is no point in changing a single driver.